tests/test_config_consistency.py

```diff
@@ -97,17 +99,67 @@ def setup_query_string_obfuscation_empty_server(self):
     @bug(context.library == "python", reason="APMAPI-772")
     def test_query_string_obfuscation_empty_server(self):
         spans = [s for _, _, s in interfaces.library.get_spans(request=self.r, full_trace=True)]
-        client_span = _get_span_by_tags(spans, tags={"http.url": "http://localhost:7777/?application_key=value"})
-        assert client_span, "\n".join([str(s) for s in spans])
+        server_span = _get_span_by_tags(spans, tags={"http.url": "http://localhost:7777/?application_key=value"})
+        assert server_span, "\n".join([str(s) for s in spans])
 
 
 @scenarios.tracing_config_nondefault
 @features.tracing_configuration_consistency
 class Test_Config_ObfuscationQueryStringRegexp_Configured:
-    def setup_query_string_obfuscation_configured(self):
+    def setup_query_string_obfuscation_configured_client(self):
+        self.r = weblog.get("/make_distant_call", params={"url": "http://weblog:7777/?ssn=123-45-6789"})
+
+    @missing_feature(context.library == "nodejs", reason="Node only obfuscates queries on the server side")
+    @missing_feature(
```

Comment:
why **test_query_string_obfuscation_empty_client** is marked as _irrrelevant_ and **test_query_string_obfuscation_configured_client** is marked as _missing_feature_?

Reply:
good catch, ty!

Reply:
should be good now